pkg/client/s3client.go

```diff
@@ -10,8 +9,12 @@ import (
 	"github.com/IBM/ibm-cos-sdk-go/service/s3/s3manager"
 	"github.com/ppc64le-cloud/pvsadm/pkg"
 	"k8s.io/klog/v2"
+
+	"fmt"
 	"os"
 	"path/filepath"
+	"regexp"
+	"strings"
 	"time"
```

Comment:
golang native libraries should be in a first section

Reply:
running it via gofmt should do the trick imho

Reply:
I am executing gofmt as well before raising PR.. anyways I have changed the code..